message_center_compassion/models/gmc_message.py

```diff
@@ -333,11 +333,16 @@ def _send_message(self, message_data):
             # Complete failure (messages were not processed)
             fail = onramp_answer.get('content', {
                 'Error': onramp_answer.get('Error', 'None')})
+            results = onramp_answer.get('content', {}).get(
+                action.connect_answer_wrapper, [])
+            error_message = str(fail.get('Error', 'None'))
             self.write({
                 'state': 'failure',
                 'failure_reason':
                     '[%s] %s' % (onramp_answer['code'],
-                                 str(fail.get('Error', 'None')))
+                                 error_message if
+                                 error_message != 'None' else
+                                 (results if results else error_message))
```

Comment:
results is not a string, it's a list but it's not a problem to display it in a message.
It's working, I tried